### ISSUE TITLE
create_result

### ISSUE THREAD

**Issue by d3orn** [d3orn/hikomsys#2]
If one of the points is 0, I divide by zero which results in an Error.
Add test to show the behavoir and fix it

**Comment by d3orn** [comment]
fixed this bug but realized I have to think about the way I give points because it should have a positive effect if a package has no dependencies and the user did not draw any. For example if you have 3 packages with only 1 dependency and you forgot about it but you knew that there are no dependencies between the other two packages you should still get a positive result I think

**Comment by mircealungu** [comment]
indeed, only if he successfully drew all the dependencies that are not in
the system, and avoided drawing all the dependencies that are in the system
should he get a 0!

right?
cheers,
m.

2014-04-12 19:30 GMT+02:00 d3orn notifications@github.com:

**Comment by d3orn** [comment]
and if he draws every possible connection he should get 0 as well
